prov/shm/src/smr_progress.c

```diff
@@ -104,6 +104,15 @@ static void smr_progress_resp(struct smr_ep *ep)
 				"unable to process tx completion\n");
 			break;
 		}
+
+		if (!ep->region->cma_cap) {
+			if (pending->map_name) {
```

Comment:
Isn't the check for map_name sufficient?  Would that ever be set if we weren't using mmap?  It's a more direct check then !cma_cap, which just means we're not using cma.

Reply:
Yeah. Updated, thank you.